src/inverseRobot.js

```diff
@@ -7,7 +7,21 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  if (typeof robot !== 'object' || robot === null || Array.isArray(robot)) {
+    throw new Error('Input must be an object');
+  }
```

Comment:
what does array.isArray() will return if we pass plain object there?

Reply:
Accordingly to the [thread](https://mateacademy.slack.com/archives/C050SHYFFR9/p1683025123681399) - I agree with your approach, this now makes sense to me 👍